cookbooks/bcpc-hadoop/recipes/hbase_repl.rb

```diff
@@ -0,0 +1,19 @@
+#
+# To enable/disable replication peers
+#
+if (node["bcpc"]["hadoop"]["hbase"]["repl"]["enabled"])
+  bash "create_hbase_repl_peer" do
```

Comment:
For the need to enable HBase replication, this simple solution would suffice since it is executed only on the HBase master node. Also this uses the components which are available by default on the ``chef-bach`` stack and maintained by HBase Apache project reducing any complexity/maintenance overhead. Tools like [HBase-Stargate](https://github.com/greglu/hbase-stargate) are good but the tool is not up to date, doesn't support  all the commands and hasn't been maintained for at least 4 years.